doc/cookbook/source/examples/pipeline/pipeline.rst

```diff
@@ -0,0 +1,29 @@
+========
+Pipeline
+========
+
+Pipeline is a machine that chains multiple transformers and machines. It consists of a sequence of transformers as intermediate stages and a machine as the final stage. Features are transformed by transformers and fed into the next stage sequentially.
```

Comment:
could you start every sentence in a new line? I.e. newline after each ".", this makes subsequent diffs way easier to review